DependencyInjection/Configuration.php

```diff
@@ -29,8 +29,8 @@ public function getConfigTreeBuilder()
                 ->arrayNode('custom_emails')
                     ->defaultValue([])
                     ->info('Customize Emails with specific role')
-                    ->arrayPrototype()
-                        ->scalarPrototype()->end()
+                    ->prototype('array')
```

Comment:
das sollte zwar in einem anderen PR sein weil es nichts mit dem test zu tun hat, aber so sieht es gut aus